config/characterizeImage.py

```diff
@@ -7,24 +7,9 @@
 
 obsConfigDir = os.path.dirname(__file__)
 
-# Cosmic rays
-# These configs match obs_subaru, to facilitate 1:1 comparisons between DECam and HSC
-config.repair.cosmicray.nCrPixelMax = 100000
-config.repair.cosmicray.cond3_fac2 = 0.4
-
 # PSF determination
 # These configs match obs_subaru, to facilitate 1:1 comparisons between DECam and HSC
 config.measurePsf.reserve.fraction = 0.2
-config.measurePsf.starSelector["objectSize"].sourceFluxField = "base_PsfFlux_instFlux"
```

Comment:
It looks like this is a change in behavior, from `base_PsfFlux_instFlux` to `base_GaussianFlux_instFlux`. Is this change expected on this ticket?

Reply:
No, PsfFlux is the new task default.